CHANGES.md

```diff
@@ -1,3 +1,19 @@
+# UNRELEASED
+
+- API:
+  - New `gc.control` type for the garbage collector settings
+  - New `gc.get` and `gc.set` for reading and writing GC settings
+  - New `gc.minor`
+  - New `gc.major`
+  - New `gc.full`
```

Comment:
`gc.compact` is missing here.

Reply:
Thanks for spotting (it is in the new API indeed)